pkg/dns/aws/dns.go

```diff
@@ -451,6 +453,7 @@ func (m *Provider) updateRecord(domain, zoneID, target, targetHostedZoneID, acti
 					ResourceRecordSet: &route53.ResourceRecordSet{
 						Name: aws.String(domain),
 						Type: aws.String(route53.RRTypeA),
+	                    TTL:  aws.Int64(ttl),
```

Comment:
@Miciah I added TTL to Alias records for consistency between the record types. Let me know if you prefer that TTL only be added to CNAME records.

Reply:
I'm seeing the following repeated in [the ingress operator logs from the last e2e-aws-operator run](https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_cluster-ingress-operator/454/pull-ci-openshift-cluster-ingress-operator-master-e2e-aws-operator/1302046117239721984/artifacts/e2e-aws-operator/pods/openshift-ingress-operator_ingress-operator-b846674d6-xl8l6_ingress-operator.log):
```
2020-09-05T01:08:17.596Z	ERROR	operator.dns_controller	dns/controller.go:181	failed to publish DNS record to zone	{"record": {"dnsName":"*.apps.ci-op-5pzpx1c2-43abb.origin-ci-int-aws.dev.rhcloud.com.","targets":["aa404373c70e04efebd0275f95643892-1078231396.us-west-2.elb.amazonaws.com"],"recordType":"CNAME","recordTTL":30}, "dnszone": {"id":"Z2GYOLTZHS5VK"}, "error": "failed to update alias in zone Z2GYOLTZHS5VK: couldn't update DNS record in zone Z2GYOLTZHS5VK: InvalidInput: Invalid request: Expected exactly one of [AliasTarget, all of [TTL, and ResourceRecords], or TrafficPolicyInstanceId], but found more than one in Change with [Action=UPSERT, Name=*.apps.ci-op-5pzpx1c2-43abb.origin-ci-int-aws.dev.rhcloud.com., Type=A, SetIdentifier=null]\n\tstatus code: 400, request id: a8d0dc74-4e7d-492b-bbb3-2d55696052ec"}
```

Reply:
@Miciah thanks for catching this. That's a similar error that required I add TTL to the CNAME record. Let me push a new commit with the TTL removed for the Alias record.